yt/visualization/geo_plot_utils.py

```diff
@@ -115,3 +115,15 @@ def get_mpl_transform(mpl_proj) -> Optional[FunctionType]:
     if key is None:
         instantiated_func = None
     return instantiated_func
+
+
+def _geographic_bounds(transform) -> tuple:
```

Comment:
could we type the tuple itself too ?

Reply:
since these functions were only used once, I just got rid of them and moved the code to where it's needed, so this type hint is gone.